docs/src/terminology.md

```diff
@@ -16,6 +20,10 @@ A front-end application that interacts with a Solana cluster.
 
 The result of interpreting all programs on the ledger at a given [tick height](terminology.md#tick-height). It includes at least the set of all [accounts](terminology.md#account) holding nonzero [native tokens](terminology.md#native-tokens).
 
+## batches
+
+To speed up the transmission of information throughout nodes, the information is cut into batches. This allows for smaller segments to be passed around, and then shared amongst peers. Once a node has received all batches, it can reconstruct the original set.
```

Comment:
This document just provides definitions. Context like "To speed up..." should be in some doc that explains the validator's implementation.

Reply:
Understood. I'm sure not all words will want to be pulled into the Solana Docs. 
I was approaching this as someone new to Solana myself, and wanting to understand how Solana is able to be so performant.